### PR TITLE
Give access to tab_id box with advanced parameters

### PR DESCRIPTION
**Pull request by alexbourret**

### CODE REVIEW

File: custom-recipes/googlesheets-append/recipe.json [hunk 1]
```diff
@@ -124,6 +124,13 @@
             ],
             "mandatory": true,
             "defaultValue": "USER_ENTERED"
+        },
```
Comment (MayeulRousselet) [inline custom-recipes/googlesheets-append/recipe.json:127]: Is it possible to change the order? For now, "Show advanced parameters" is the last parameter. I will expect "Sheet name (legacy)" to appears/hide below, not above.

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]
The behaviour of the "Show advanced parameters" is a bit strange for existing recipes. Clicking on it does nothing if "Sheet name (legacy)" has a value. Maybe we could rename the label to something more consistent?

File: custom-recipes/googlesheets-append/recipe.json (hunk 1 above)
Comment (alexbourret) [reply custom-recipes/googlesheets-append/recipe.json:127]: But when you click on the recipe of a legacy project, that box appears just above the preferred new selector, with the description "Please use the selector below"

Comment (MayeulRousselet) [reply custom-recipes/googlesheets-append/recipe.json:127]: I've jut understand what you mean: the help text is not clear, we should replace it by "Legacy, please prefer using the Sheet name selector below."

Comment (MayeulRousselet) [reply custom-recipes/googlesheets-append/recipe.json:127]: Done in [a1e89a5](https://github.com/dataiku/dss-plugin-googlesheets/pull/9/commits/a1e89a564acc3b406c65b141692f98a5b69e1b7a)

**Review by MayeulRousselet — APPROVED** [review]

**Pull request merged**